src/testUtils/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { mount, shallow, ReactWrapper, ShallowWrapper } from 'enzyme';
+import * as React from 'react';
+
+export interface Resp {
+    results: Array<{ [key: string]: any }>;
+    total?: number;
+}
+
+export interface QueryString {
+    [key: string]: string;
+}
+
+/**
+ * Compose setup method for component tests
+ */
+export const createSetup = <P extends {}, C extends {} = { [contextType: string]: any }>(
```

Comment:
i noticed the use of a 'setup' method [here](https://redux.js.org/recipes/writing-tests) and [here](https://medium.com/selleo/testing-react-components-best-practices-2f77ac302d12), but figured making this a higher-order function is more dry (alternative being definition a 'setup' method in each test file). we also need to be able to pass/read from context.